index.php

```diff
@@ -31,6 +31,11 @@
 $settingsResponse = json_decode($values);
 $values = $l->suggest('fossasia', 10, 'asc');
 $suggestResponse = json_decode($values);
+$values = $l->aggregations("fossasia", "", "", array("mentions","hashtags"), 10);
+// Some other possible inputs
+// $l->aggregations("fossasia", "2015-01-10", "2015-10-21", array("mentions","hashtags"), 10);
+// $l->aggregations("fossasia", "", "", array("mentions","hashtags"));
```

Comment:
Should a single `field` value also be sent in an `array()` notation ? It looks like the code is using a `foreach`, This maynot be the best thing that we could do right ? What are the other ways in which we could design this ?